replay/models/als.py

```diff
@@ -275,7 +275,7 @@ def _filter_seen(
         """
 
         if not self._hnswlib_params:
-            return Recommender._filter_seen(self, recs, log, k, users)
+            return super()._filter_seen(recs, log, k, users)
```

Comment:
Code duplication  in ALS and Word2Vec. _filter_seen should be defined in ANNMixin with this codebase. No need to redefine method in individual classes.